tools/init_databases.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+# Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+# For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+# import required module
+import os
+import sqlite3
+import argparse
+
+
+# this helper script creates DBs from SQL schema files
+
+def process_file(path: os.path, dst_directory: os.path) -> None:
+    file_name = os.path.basename(path)
+    print(file_name)
+
+    db_name = file_name.split("_0")[0]
+    db_name = db_name + ".db"
+    dst_db_path = os.path.join(dst_directory, db_name)
+    print(f"Database name: {db_name}\n")
+    print(f"Executing {path} script into {dst_db_path} database")
+
+    try:
+        connection = sqlite3.connect(dst_db_path)
+        with open(path) as fp:
+            connection.executescript(fp.read())
+        connection.commit()
+    except OSError as e:
+        print(e)
+    finally:
+        if connection:
+            connection.close()
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Create databases from schema scripts')
```

Comment:
There is pattern among python users to define `main` function and run only main method in `if __name__ = "__main__" ` section.

It gives the advantage of returning errors in C-like style (`sys.exit(main())`)
Reference here
https://docs.python.org/3/library/__main__.html#idiomatic-usage

Reply:
added main function + return codes

Reply:
Ok